bundles/action/org.openhab.action.mail/README.md

```diff
@@ -1,14 +1,16 @@
 # Mail Actions
 
-This add-on provides SMTP services so your rules and scripts can send e-mails.
-The `to` paremeter can contain a semicolon-separated list of email addresses.
+This add-on provides SMTP services so your rules and scripts can send emails.
+
 
 ## Actions
 
 - `sendMail(String to, String subject, String message)`: Sends an email via SMTP.
 - `sendMail(String to, String subject, String message, String attachmentUrl)`: Sends an email with attachment via SMTP.
 - `sendMail(String to, String subject, String message, List<String> attachmentUrlList)`: Sends an email with one or more attachments via SMTP.  
 
+The `to` parameter can contain a semicolon-separated list of email addresses. Email addresses can be either in the format "user@domain.com" or "First Lastname <user@domain.com>". 

```

Comment:
"Email addresses can be specified in one of these formats: ..."

Also, you have to escape the left-angle bracket for it to appear properly.  This can be done by prefixing it with a backslash.

Reply:
Edited - please check and resolve if okay.